pkgs/by-name/fl/flashprog/package.nix

```diff
@@ -1,49 +1,63 @@
-{ fetchgit
+{ fetchurl
 , installShellFiles
 , lib
 , libftdi1
-, libgpiod_1
+, libgpiod
 , libjaylink
 , libusb1
 , pciutils
 , pkg-config
 , stdenv
+, with_jlink ? true
+, with_gpio ? true
 }:
 
 stdenv.mkDerivation rec {
   pname = "flashprog";
   version = "1.0.1";
 
-  src = fetchgit {
-    url = "https://review.sourcearcade.org/flashprog";
-    rev = "2ca11f9a4101ea230081d448ab2b570425b7f0bd";
-    hash = "sha256-pm9g9iOJAKnzzY9couzt8RmqZFbIpKcO++zsUJ9o49U=";
+  src = fetchurl {
+    url = "https://flashprog.org/releases/flashprog-v${version}.tar.bz2";
+    hash = "sha256-iIhj38JUqd18AN/ayKMadIecMdtcFjdfs/IWNHwDzHE=";
   };
 
-  nativeBuildInputs = [
-    installShellFiles
-    pkg-config
-  ];
+  nativeBuildInputs = [ pkg-config installShellFiles ];
+  buildInputs = [ libftdi1 libusb1 ]
+    ++ lib.optionals (!stdenv.isDarwin) [ pciutils ]
+    ++ lib.optionals with_jlink [ libjaylink ]
+    ++ lib.optionals (stdenv.isLinux && with_gpio) [ libgpiod ];
+
+  postPatch = ''
+    substituteInPlace util/z60_flashprog.rules \
+      --replace 'GROUP="plugdev"' 'TAG+="uaccess", TAG+="udev-acl"'
+  '';
+
+  makeTargets = [ "libinstall" ];
 
-  buildInputs = [
-    libftdi1
-    libjaylink
-    libusb1
-  ] ++ lib.optionals (!stdenv.isDarwin) [
-    libgpiod_1
-    pciutils
+  makeFlags = [
+    "PREFIX=$(out)"
+  ] ++ lib.optionals (!with_jlink) [
+    "CONFIG_JLINK_SPI=no"
+  ] ++ lib.optionals (!(stdenv.isLinux && with_gpio)) [
+    "CONFIG_LINUX_GPIO_SPI=no"
+  ] ++ lib.optionals stdenv.isDarwin [
+    "CONFIG_ENABLE_LIBPCI_PROGRAMMERS=no"
+  ] ++ lib.optionals (stdenv.isDarwin && stdenv.isx86_64) [
+    "CONFIG_INTERNAL_X86=no"
+    "CONFIG_INTERNAL_DMI=no"
+    "CONFIG_RAYER_SPI=no"
   ];
 
-  makeFlags = [ "PREFIX=$(out)" "libinstall" ]
-    ++ lib.optionals stdenv.isDarwin [ "CONFIG_ENABLE_LIBPCI_PROGRAMMERS=no" ]
-    ++ lib.optionals (stdenv.isDarwin && stdenv.isx86_64) [ "CONFIG_INTERNAL_X86=no" "CONFIG_INTERNAL_DMI=no" "CONFIG_RAYER_SPI=0" ];
+  postInstall = ''
+    install -Dm644 util/z60_flashprog.rules $out/lib/udev/rules.d/z60_flashprog.rules
+  '';
 
-  meta = with lib; {
+  meta = {
     homepage = "https://flashprog.org";
     description = "Utility for reading, writing, erasing and verifying flash ROM chips";
-    license = with licenses; [ gpl2 gpl2Plus ];
-    maintainers = with maintainers; [ felixsinger ];
-    platforms = platforms.all;
+    license = with lib.licenses; [ gpl2Plus ];
     mainProgram = "flashprog";
+    maintainers = with lib.maintainers; [ felixsinger funkeleinhorn ];
```

Comment:
I appreciate your interest in becoming a maintainer for this package. Though, maintainers should work well together, so that conflicts are avoided. I'm open and looking forward to work with you in the future, but for now I rather would like to wait with the maintainership here. If you would like to, let's have a chat.

Reply:
Okayy. Also looking forward to work with you in the future. I removed myself from the maintainers list for now. I would be open to have a chat.